docs/extend/extend-apiml/api-mediation-oidc-authentication.md

```diff
@@ -164,4 +146,14 @@ The OIDC provider returns an HTTP 40x error code.
 The client application is not properly configured in the API ML Gateway.
 
 **Solution**  
-Check that the `client_id` and `client_secret` configured in the API ML Gateway correspond to the `client_id` and `client_secret` of the client application as configured in the OIDC provider.
+Check that the URL `jwks_uri` contain actual key for the OIDC token validation.
+
+
+## Tips
+API ML Gateway exposes validate token `POST /gateway/api/v1/auth/oidc-token/validate` operation which is suitable during the OIDC setup. It expects JSON `{ token, serviceId }`. Call allows to verify that OIDC token is trusted by the API ML. Accounts mapping step is not included in that flow. 
+
+## Azure Entra ID OIDC notes
+At the current version API ML uses `sub` claim of the ID Token is used to identify the user and map to MF account. [Azure ID token](https://learn.microsoft.com/en-us/entra/identity-platform/id-token-claims-reference#use-claims-to-reliably-identify-a-user) contains several user identifiers. Azure token `sub` is unique per each app alphanumeric value e.g. `vkDzPtDkdH1qjI9h1N1dcEqCkhVASuXLqMDtykPZhC0` whereas OKTA ID token has email in `sub` claim. So z/OS identity propagation setup should take this into account for proper account mapping configuration. See Azure clarification:  
```

Comment:
What is "the current version API ML"? Please be more specific. I do not understand the structure of this sentence. 
* Define what is the sub claim.
* Clearly describe the difference between the Azure token sub and the OKTA ID token.
* Provide an example in z/OS identity propagation setup of how the Azure token sub and the OKTA ID token are configured.
* Remove the sentence "So z/OS identity propagation setup should take this into account for proper account mapping configuration". This is too vague.
* Remove "See Azure clarification". If there is a document that explains Azure, reference the title of the document without a link. 
Please re-write.

Reply:
I main statement here is that Azure OIDC token differ from OKTA OIDC token and exactly field which we use for account mapping has different logical value in there. I put reference to MS documents where all this described. No citation anymore.